GoMain/src/main/main.go

```diff
@@ -161,5 +165,28 @@ func orchestrationInit() error {
 
 	log.Println(logPrefix, "orchestration init done")
 
+	if isSecured {
+		mnedcmgr.GetServerInstance().SetCipher(dummy.GetCipher(cipherKeyFilePath))
+		mnedcmgr.GetServerInstance().SetCertificateFilePath(certificateFilePath)
```

Comment:
Yes, because in those packages, TCP server/client are being created and they need to know if it has be secured or not.
If SetCertificateFilePath() is called, secure client/server will be created, else it will be unsecured.

Reply:
@t25kim I applied the change. Now Client instance wont be called if a device is not a client. Same applied for server. You can ignore my previous comment, I misunderstood your point.